### DISCUSSION TITLE
How to create indented sidebar navigation?

### DISCUSSION THREAD

**Discussion by couleurm** [squidfunk/mkdocs-material#6460] [Q&A] [answered]
Hey, is there some config / css trick to indent child pages?

![image](https://github.com/squidfunk/mkdocs-material/assets/82747632/e9ee32e8-2a10-441b-8988-fbaa1bef7fdf)

**Comment by kamilkrzyskow** [comment]
Hmm, they should be indented by default if they're really child pages of a section. 
You can even see the indentation on your public website:
https://ctt.cx/video/

<details>
<summary>Screenshot</summary>

![image](https://github.com/squidfunk/mkdocs-material/assets/34622465/06b88bc2-1cd1-486d-bb52-0557110ccb34)

</details>

---

I would need a minimal reproduction of some sort to be able to check if there are any css tricks available 🤔 
https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/

**Comment by couleurm** [comment]
They're indented when there is that  ![image](https://github.com/squidfunk/mkdocs-material/assets/82747632/d20fd76f-c8f2-48d0-9c07-e477b62d0424) button is available, that shows on the child pages' child pages, not simple child pages


[mkdocs-glightbox](https://blueswen.github.io/mkdocs-glightbox/) have their in-page headings look up the exact way i'd love having my child pages set up 

![image](https://github.com/squidfunk/mkdocs-material/assets/82747632/232aea3a-faf5-46cb-bda1-e849bd5260d6)

edit: that's `features: toc.integrate` doing this 

There's probably some css trickery to steal from that, I'd appreciate if a css wizard found a way 👁👁

**Reply by kamilkrzyskow** [reply, marked as answer]
Since you've updated the public version to match the initial screenshot, I could look at the source.
This should fit your needs:
```css
.md-nav[data-md-level="2"] > .md-nav__list {
  margin-left: 1em;
  border-left: 1px solid var(--md-accent-fg-color);
  padding-left: 0.5em;
}
```
<details>
<summary>Screenshoot</summary>

![image](https://github.com/squidfunk/mkdocs-material/assets/34622465/b5df19d0-adc2-4189-a4c4-a2458900b3ae)

</details>

I didn't look at the `toc.intergrate` CSS creating this, so perhaps there might be something better 🤷
